From 39a10d32f97a1f021e333fa5f49cad998c0e23e7 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Michal=20=C4=8Ciha=C5=99?= <michal@cihar.com>
Date: Thu, 10 Sep 2020 08:45:05 +0200
Subject: [PATCH 1/1] Formats: Compatibility with upcoming translate-toolkit

---
 weblate/formats/ttkit.py | 8 +++++++-
 1 file changed, 7 insertions(+), 1 deletion(-)

diff --git weblate/formats/ttkit.py weblate/formats/ttkit.py
index 77c48e8c9f..7809e22330 100644
--- weblate/formats/ttkit.py
+++ weblate/formats/ttkit.py
@@ -1297,7 +1297,13 @@ class DTDFormat(TTKitFormat):
     @property
     def all_store_units(self):
         """Wrapper for all store unit filtering out null."""
-        return (unit for unit in self.store.units if not unit.isnull())
+        # The getattr is needed for translate-toolkit 3.0 and 3.1,
+        # once support for 3.0 is dropped, isblank can be used
+        return (
+            unit
+            for unit in self.store.units
+            if not getattr(unit, "isnull", unit.isblank)()
+        )
 
 
 class SubtitleUnit(MonolingualIDUnit):
-- 
2.28.0

