From 3e6478dd999f91453014c2795379af1cfa2a88fc Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Michal=20=C4=8Ciha=C5=99?= <michal@cihar.com>
Date: Mon, 15 Jul 2019 16:05:07 +0200
Subject: [PATCH 1/1] Compatibility with rest-framework 3.10.0
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Signed-off-by: Michal Čihař <michal@cihar.com>
---
 weblate/auth/admin.py | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/weblate/auth/admin.py b/weblate/auth/admin.py
index 049b560fbe..13dd4b0c59 100644
--- a/weblate/auth/admin.py
+++ b/weblate/auth/admin.py
@@ -181,3 +181,8 @@ class WeblateGroupAdmin(WeblateModelAdmin):
             request, form, formsets, change
         )
         self.new_obj.save()
+
+
+# This is pointless for Weblate, but necessary to silent admin.E039
+# check when using custom admin site
+admin.site.register(User, WeblateUserAdmin)
-- 
2.20.1

