From 45ddceedb66b256f75f491c2fe42ee4bd5b10980 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Michal=20=C4=8Ciha=C5=99?= <michal@cihar.com>
Date: Mon, 15 May 2023 11:22:53 +0200
Subject: [PATCH 1/1] legal: Add privacy URL to TOS confirmation

---
 weblate/legal/views.py | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git weblate/legal/views.py weblate/legal/views.py
index 052e22acee..025e5eb078 100644
--- weblate/legal/views.py
+++ weblate/legal/views.py
@@ -77,4 +77,8 @@ def tos_confirm(request):
     else:
         form = TOSForm(initial={"next": request.GET.get("next")})
 
-    return render(request, "legal/confirm.html", {"form": form})
+    return render(
+        request,
+        "legal/confirm.html",
+        {"form": form, "privacy_url": reverse("legal:privacy")},
+    )
-- 
2.32.0

