From d12ba86438ae2f16a45384f3a40ccbd08461dc31 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Michal=20=C4=8Ciha=C5=99?= <michal@cihar.com>
Date: Tue, 16 Jun 2020 10:50:49 +0200
Subject: [PATCH 1/1] Component: Remove debugging code

---
 weblate/trans/models/component.py | 1 -
 1 file changed, 1 deletion(-)

diff --git weblate/trans/models/component.py weblate/trans/models/component.py
index a62d98f2fd..5259b16101 100644
--- weblate/trans/models/component.py
+++ weblate/trans/models/component.py
@@ -740,7 +740,6 @@ class Component(FastDeleteMixin, models.Model, URLMixin, PathMixin):
 
     @cached_property
     def background_task(self):
-        print(self.__dict__)
         task_id = cache.get(self.update_key)
         if not task_id:
             return None
-- 
2.27.0

