From 53b3e55e1fd7f27c3ec0d9188eff61f92692119d Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Michal=20=C4=8Ciha=C5=99?= <michal@cihar.com>
Date: Fri, 5 Jul 2019 10:15:23 +0200
Subject: [PATCH 1/1] Migration: Avoid crash on multiple plurals
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Signed-off-by: Michal Čihař <michal@cihar.com>
---
 weblate/lang/models.py | 29 ++++++++++++++++-------------
 1 file changed, 16 insertions(+), 13 deletions(-)

diff --git a/weblate/lang/models.py b/weblate/lang/models.py
index 2f60bb705f..777abbfaf6 100644
--- a/weblate/lang/models.py
+++ b/weblate/lang/models.py
@@ -303,19 +303,22 @@ class LanguageQuerySet(models.QuerySet):
                 'number': nplurals,
                 'equation': pluraleq,
             }
-            plural, created = lang.plural_set.get_or_create(
-                source=Plural.SOURCE_DEFAULT,
-                language=lang,
-                defaults=plural_data,
-            )
-            if not created:
-                modified = False
-                for item in plural_data:
-                    if getattr(plural, item) != plural_data[item]:
-                        modified = True
-                        setattr(plural, item, plural_data[item])
-                if modified:
-                    plural.save()
+            try:
+                plural, created = lang.plural_set.get_or_create(
+                    source=Plural.SOURCE_DEFAULT,
+                    language=lang,
+                    defaults=plural_data,
+                )
+                if not created:
+                    modified = False
+                    for item in plural_data:
+                        if getattr(plural, item) != plural_data[item]:
+                            modified = True
+                            setattr(plural, item, plural_data[item])
+                    if modified:
+                        plural.save()
+            except Plural.MultipleObjectsReturned:
+                continue
 
         # Create addditiona plurals
         for code, dummy, nplurals, pluraleq in languages.EXTRAPLURALS:
-- 
2.20.1

