From 8e9645d4202cd3e39a7bb5bb43630d8060b22bd6 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Michal=20=C4=8Ciha=C5=99?= <michal@cihar.com>
Date: Sun, 14 May 2023 19:35:36 +0200
Subject: [PATCH 1/1] docker: Add MusicBrainzOAuth2

---
 weblate/settings_docker.py | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git weblate/settings_docker.py weblate/settings_docker.py
index 1c90afc958..da092d6146 100644
--- weblate/settings_docker.py
+++ weblate/settings_docker.py
@@ -338,6 +338,13 @@ SOCIAL_AUTH_GOOGLE_OAUTH2_WHITELISTED_EMAILS = get_env_list(
     "WEBLATE_SOCIAL_AUTH_GOOGLE_OAUTH2_WHITELISTED_EMAILS"
 )
 
+SOCIAL_AUTH_MUSICBRAINZ_KEY = os.environ.get("WEBLATE_SOCIAL_AUTH_MUSICBRAINZ_KEY", "")
+SOCIAL_AUTH_MUSICBRAINZ_SECRET = os.environ.get(
+    "WEBLATE_SOCIAL_AUTH_MUSICBRAINZ_SECRET", ""
+)
+if SOCIAL_AUTH_MUSICBRAINZ_KEY and SOCIAL_AUTH_MUSICBRAINZ_SECRET:
+    AUTHENTICATION_BACKENDS += ("social_core.backends.musicbrainz.MusicBrainzOAuth2",)
+
 if "WEBLATE_SOCIAL_AUTH_GITLAB_KEY" in os.environ:
     AUTHENTICATION_BACKENDS += ("social_core.backends.gitlab.GitLabOAuth2",)
 
-- 
2.39.2

