From 4d8d75dd99e6b2fd64ec2f31aef6f42e1daff1f2 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Michal=20=C4=8Ciha=C5=99?= <michal@cihar.com>
Date: Fri, 12 May 2023 20:32:44 +0200
Subject: [PATCH 1/1] docker: Extend SAML configuration

---
 weblate/settings_docker.py | 12 +++++++++---
 1 file changed, 9 insertions(+), 3 deletions(-)

diff --git weblate/settings_docker.py weblate/settings_docker.py
index 2196eb116f..1c90afc958 100644
--- weblate/settings_docker.py
+++ weblate/settings_docker.py
@@ -377,9 +377,15 @@ if "WEBLATE_SAML_IDP_URL" in os.environ:
             "entity_id": os.environ.get("WEBLATE_SAML_IDP_ENTITY_ID", ""),
             "url": os.environ.get("WEBLATE_SAML_IDP_URL", ""),
             "x509cert": os.environ.get("WEBLATE_SAML_IDP_X509CERT", ""),
-            "attr_name": "full_name",
-            "attr_username": "username",
-            "attr_email": "email",
+            "attr_name": os.environ.get("WEBLATE_SAML_ID_ATTR_NAME", "full_name"),
+            "attr_username": os.environ.get(
+                "WEBLATE_SAML_ID_ATTR_USERNAME", "username"
+            ),
+            "attr_email": os.environ.get("WEBLATE_SAML_ID_ATTR_EMAIL", "email"),
+            "attr_user_permanent_id": os.environ.get(
+                "WEBLATE_SAML_ID_ATTR_USER_PERMANENT_ID",
+                "urn:oid:0.9.2342.19200300.100.1.1",
+            ),
         }
     }
     SOCIAL_AUTH_SAML_SUPPORT_CONTACT = SOCIAL_AUTH_SAML_TECHNICAL_CONTACT = {
-- 
2.39.2

