From 6bc09c6b1599542d133d6e79611ba3fcc7490276 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Michal=20=C4=8Ciha=C5=99?= <michal@cihar.com>
Date: Mon, 27 Apr 2020 22:52:20 +0200
Subject: [PATCH 2/2] Utils: Fix triggering weblate.E003 check

---
 weblate/utils/checks.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git weblate/utils/checks.py weblate/utils/checks.py
index ecf3c802ad..764f70987a 100644
--- weblate/utils/checks.py
+++ weblate/utils/checks.py
@@ -118,7 +118,7 @@ def check_mail_connection(app_configs, **kwargs):
         connection.close()
     except Exception as error:
         message = "Cannot send e-mail ({}), please check EMAIL_* settings."
-        errors.append("weblate.E003", weblate_check(message.format(error)))
+        errors.append(weblate_check("weblate.E003", message.format(error)))
 
     return errors
 
-- 
2.26.2

