From ddd1eda0125884e3901b852e83d9c0c01f91ec3d Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Michal=20=C4=8Ciha=C5=99?= <michal@cihar.com>
Date: Fri, 29 Jun 2018 14:23:09 +0200
Subject: [PATCH 1/1] Invalidate per project cache on start
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Signed-off-by: Michal Čihař <michal@cihar.com>
---
 weblate/trans/tests/test_views.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/weblate/trans/tests/test_views.py b/weblate/trans/tests/test_views.py
index be1792651..cc952d422 100644
--- a/weblate/trans/tests/test_views.py
+++ b/weblate/trans/tests/test_views.py
@@ -90,6 +90,7 @@ class ViewTestCase(RepoTestCase):
         self.component = self.create_component()
         self.project = self.component.project
         # Invalidate caches
+        self.project.stats.invalidate()
         cache.clear()
         # Login
         self.client.login(username='testuser', password='testpassword')
-- 
2.17.1

